dawn/renderer/utils/sagas.js

```diff
@@ -248,12 +253,13 @@ function* ansibleGamepads() {
     // navigator.getGamepads always returns a reference to the same object. This
     // confuses redux, so we use assignIn to clone to a new object each time.
     const newGamepads = _.assignIn({}, navigator.getGamepads());
-    if (_needToUpdate(newGamepads)) {
+    if (_needToUpdate(newGamepads) || Date.now() - timestamp > 1000) {
```

Comment:
Shouldn't be affecting anything. Current implementation was to already send whenever gamepad information changes (i.e. any movement detected) and set a 50ms cooldown.